client/ws-client/src/lib.rs

```diff
@@ -216,11 +216,7 @@ impl WsClientBuilder {
 	}
 
 	/// Build the [`WsClient`] with specified [`TransportSenderT`] [`TransportReceiverT`] parameters
-	///
-	/// ## Panics
```

Comment:
nit: Wouldn't this panic? (build_with_stream and build are having the panic section)